seminar_4/task_8.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+import os
+import time
+from pathlib import Path
+import requests
+import threading
+import multiprocessing
+import asyncio
+
+image_urls = []
+with open('images.txt', 'r') as images:
+    for image in images.readlines():
+        image_urls.append(image.strip())
+
```

Comment:
Но можно сделать вообще иначе.
```suggestion
image_urls = []
with open('images.txt', 'r') as images:
    image_urls = images.read().split('\n') 

```